docker/README.md

```diff
@@ -21,25 +21,29 @@ In a directory that contains the `Dockerfile` included in this repository, on a
 
 ### Upgrade Process Recommendations
 
-The FiftyOne Teams 0.8.8 Database (version `0.16.6`) is forward-compatible with the FiftyOne Teams 0.9.2 Client (database version `0.17.2`).  Voxel51 recommends the following upgrade process:
+The FiftyOne Teams 0.10.0 Client (database version `0.18.0`) is backwards-compatible with the FiftyOne Teams 0.8.8 Database (version `0.16.6`) and all the versions between. Voxel51 recommends the following upgrade process:
 
 1. Ensure all Python clients set `FIFTYONE_DATABASE_ADMIN=false` (this should generally be your default)
-1. Upgrade FiftyOne Teams Python clients to FiftyOne Teams v0.9.2
-1. Upgrade your FiftyOne Teams App deploy to version v0.2.2
+1. Upgrade FiftyOne Teams Python clients to FiftyOne Teams v0.10.0
```

Comment:
Why all step 1?

Reply:
that's how GitHub Markdown makes a numbered list - it looks like [this](https://helm.fiftyone.ai/) in real life

Reply:
Well I'll be darned; learned something today, ty

Reply:
...and it's not even 10AM! 🥳